meta-lmp-base/recipes-extended/ostree/ostree/0006-Add-support-systemd-boot-automatic-boot-assesment.patch

```diff
@@ -0,0 +1,148 @@
+From a9e35b76fd5e47b9bc29a7113d97a4d98a973e75 Mon Sep 17 00:00:00 2001
+From: Igor Opaniuk <igor.opaniuk@foundries.io>
+Date: Wed, 11 Sep 2024 18:03:10 +0200
+Subject: [PATCH] Add support systemd-boot automatic boot assesment
+
+Add support for Automatic Boot Assessment [1].
+Boot entries are now created with an additional suffix, which
+represents the amount of maximum tries for boot counting.
+
+[1] https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/
+Upstream-Status: Inappropriate [lmp specific]
```

Comment:
We should probably set as pending here, as this is something we want to try to upstream once there is support for systemd.